website/static/js/commentpane.js

```diff
@@ -11,7 +11,10 @@
         onOpen: function() {}
     };
 
-    var CommentPane = CommentPane || function(selector, options) {
+    /*
+       mode: 'widget', 'pane' or 'page'
+     */
+    var CommentPane = CommentPane || function(selector, hostPage, hostName, mode, options) {
```

Comment:
@sloria I removed hostpage and hostname. The <code>mode</code> is REALLY USEFUL in this module when resizing.
This is what happens when I delete that field:

![screen shot 2015-04-02 at 10 33 14 am](https://cloud.githubusercontent.com/assets/8628591/6967806/dc2afbe0-d92f-11e4-8b34-9eedc16f0cd9.png)

On resize... (making it wider)
![screen shot 2015-04-02 at 10 33 31 am](https://cloud.githubusercontent.com/assets/8628591/6967820/f70dcb5e-d92f-11e4-8ddb-87b952bd2796.png)